config/application.rb

```diff
@@ -17,5 +17,7 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+
+    Dor::WorkflowService.configure(Settings.workflow_url, { 'logger': Logger.new(Settings.workflow_services_log) } )
```

Comment:
wondering if the pattern is for this to live here, or elsewhere (initializer?  other?)

Reply:
a `config/initializers` file sounds like the right home to me.